data/human/navy missions intro.txt

```diff
@@ -0,0 +1,1496 @@
+		fleet "Large Northern Pirates"
+
+	npc
+		government "Pirate"
+		personality nemesis entering harvests plunders
+		system "Acamar"
+		fleet "Large Northern Pirates"
+	npc
+		government "Pirate"
+		personality nemesis staying harvests plunders
+		system "Cardax"
+		fleet "Small Northern Pirates"
+		fleet "Large Northern Pirates"
+	
+	npc accompany save
+		government "Republic"
+		personality escort timid
+		fleet
+			names "republic" "capital"
+			variant
+				"Auxiliary (Cargo)" 4
+
+	on stopover
+		event "geminus phase 1"
+		conversation
+			`Your clearance to land on Lodestone was delayed by volcanic ash for about ten minutes before the port authorities allowed your fleet to land. "Fourth Port," as the spaceport is called, was built after the previous three ports were buried under Lodestone's numerous eruptions. Unlike your previous stay at <source>, where you would have struggled to have found a speck of dust in the mess hall, Fourth Port is a testament to the Syndicate's standards: warehouses of rusted sheetmetal, towering metal smokestacks that belching caustic ash, and barrels containing a disgusting yellowish liquid you can only hope are cleaning chemicals and not...something else.`
+			`	 Landing in the hangar designated for your fleet doesn't do the Syndicate's image any favors, either. Workers are actively repairing parts of the walls and ceiling that you felt should've been replaced long ago, and the rickety scaffolding they've erected for the repair crews makes you worried, to say the least. The air in the hangar is also thick and stagnant, and almost makes you gag. You quickly call over the captains of the auxiliaries and make your way to the loading docks - it takes about ten minutes to make your way through the maze of corridors before you reach your destination.`
+			`	Upon reaching the docks, several Syndicate dockworkers call you over to a large collection of cargo containers filled with the building materials bound for Geminus. "Took you long enough," one says with a cigarette hanging out the side of his mouth. "We've been waiting here for five hours with our thumbs up our collective asses for you Navy jarheads. One of your own came by here about a month ago and acted like they owned the place - I tell ya, my boss doesn't pay me enough to put up with shit like theirs."`
+			choice
+				`	"Sorry to have kept you waiting. We came as quickly as we could."`
+					goto end
+				`	"You better keep that tongue of yours in check, my man, or you'll be choking on that cigarette."`
+					goto fight
+				`	"Hold on. 'One of our own?' You mean someone from the Navy?"`
+					goto investigate
+			label fight
+			apply
+				set "bad look"
+			`The dockworker chuckles for a second, takes his cigarette out of his mouth and flicks it at you. You almost throw a punch before one of the captains of the auxiliaries steps between the two of you. "Sir, it's been a long day for all of us. Could you please help us load our cargo onto our ships?"`
```

Comment:
This is a strong action on <first>'s part. Even if you're going to get stopped, the decision to want to throw a punch probably shouldn't be narration.